.snyk

```diff
@@ -0,0 +1,8 @@
+# References:
```

Comment:
This commit could be also squashed into `UPSTREAM: <carry>: Add OpenShift files`